### ISSUE TITLE
provide locations via android mock locations developer API

### ISSUE THREAD

**Issue by hhm0** [microg/android_packages_apps_UnifiedNlp#27]
Thanks for the onderful project! :-)

In android, there is an option in the Developer options to "enable mock locations". It would be wonderful to have this implemented, as a straightforward option for providing the Android system with location data.

(see <a href="http://blog.mapillary.com/technology/2014/10/01/bt-gps.html">here</a> for example of using this API for injecting location data).

**Comment by mar-v-in** [comment]
A lot of apps do not work properly with the mock locations, this happens whenever a location provider is selected directly instead of automatically. That's why the network location provider api was choosen to implement this.

However I agree that using the mock API might become an interesting additional option.

**Comment by hhm0** [comment]
Yes, i mainly meant having it as another alternative for location supply :-)

**Comment by brandynicole1018** [comment]
<?xml version="1.0" encoding="utf-8"?>
<manifest
package="org.gfd.gsmlocation"
xmlns:android="http://schemas.android.com/apk/res/android"
android:versionCode="15"
android:versionName="0.0.1.5"
>

**Comment by KUKHUA** [comment]
:(
